docs/source/_extensions/_static/accordian.css

```diff
@@ -0,0 +1,33 @@
+/* Accordians */
```

Comment:
```suggestion
/* Accordions */
```
Probably also adjust the filename